### PR TITLE
Create Home Page, which displays the most recent products and welcomes the user.

### PR DESCRIPTION
**Pull request by jwq1**
This pull request is to create an MVP home page. It displays the two most recently added products and welcomes the user by their username.

### CODE REVIEW

File: final-project/application.py [hunk 1]
```diff
@@ -41,10 +41,43 @@ def index():
...
+
+    # Get username of the current user
+    username = user_info[0]['username']
+
+    # Get the most recently added products. Limit to 2.
+    recent_products = db.execute(
+            "SELECT product_name, image"
+            + " FROM products"
+            + " ORDER BY id DESC"
+            + " LIMIT 2")
```
Comment (hdavidzhu) [inline final-project/application.py]: Much better! There are also these ways of making multi-line string: https://stackoverflow.com/a/10660443/2204868

File: final-project/application.py [hunk 2]
```diff
@@ -41,10 +41,43 @@ def index():
...
+
+    # Ensure recent products were found.
+    if recent_products == None:
+        return apology("No recent products")
+
+    # Count products returned
+    product_count = 0
+    for rows in recent_products:
+        product_count += 1
```
Comment (hdavidzhu) [inline final-project/application.py]: Try some array comprehensions here for more brevity. I think you can work in a `len(...)` somewhere in here.

**Review by hdavidzhu — COMMENTED** [review]
Much better in terms of PR scope! Feel free to make the PR title longer, and to use PR as a abbr for pull requests in the future.

File: final-project/application.py (hunk 1 above)
Comment (jwq1) [reply final-project/application.py]: thanks, I read through the whole python style guide and didn't come across this. I am glad you could find it in a couple seconds haha.

Those multi-line strings took way too long to make in my version. I'll change it now.

**Pull request merged**